apps/emqx_management/src/emqx_mgmt_api_data.erl

```diff
@@ -162,7 +162,7 @@ import_content(Content) ->
 tmp_filename() ->
     Seconds = erlang:system_time(second),
     {{Y, M, D}, {H, MM, S}} = emqx_mgmt_util:datetime(Seconds),
-    list_to_binary(io_lib:format("emqx-export-~p-~p-~p-~p-~p-~p.json", [Y, M, D, H, MM, S])).
+    iolist_to_binary(io_lib:format("emqx-export-~p-~p-~p-~p-~p-~p.json", [Y, M, D, H, MM, S])).
```

Comment:
This one doesn't need `unicode:characters_to_binary`?

Reply:
This is certainly going to be latin1